mmv1/third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -298,6 +298,33 @@ func TestAccComputeInstanceTemplate_preemptible(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstanceTemplate_maintenance_interval(t *testing.T) {
+	t.Parallel()
+
+	var instanceTemplate compute.InstanceTemplate
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeInstanceTemplateDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
Can you please add steps to remove the field maintenance_interval?

Reply:
I added one more step but not sure if that is what you meant

Reply:
It is. Thanks.